### PR TITLE
Cs 5608 clear template on playlist update

### PR DESCRIPTION
**Pull request by ahilles107**

### CODE REVIEW

File: newscoop/application/modules/admin/controllers/ThemesController.php [hunk 1]
```diff
@@ -311,6 +311,36 @@ public function wizardThemeSettingsAction()
         $this->view->themeForm = $themeForm;
     }
 
+    public function wizardThemePlaylistsAction()
+    {
+        $playlistsService = \Zend_Registry::get('container')->getService('playlists');
+        $theme = $this->getThemeService()->findById($this->_request->getParam('id'));
+        $path = __DIR__.'/../../../../themes/'.$theme->getPath().'theme.xml';
```
Comment (takeit) [inline newscoop/application/modules/admin/controllers/ThemesController.php]: you can use `ThemesService` to get theme path

Comment (ahilles107) [inline newscoop/application/modules/admin/controllers/ThemesController.php]: will check if fetching new service will be faster here.

Comment (takeit) [inline newscoop/application/modules/admin/controllers/ThemesController.php]: ahh you can ignore my previous comment, you are getting specific theme

File: newscoop/library/Newscoop/Services/PlaylistsService.php [hunk 2]
```diff
@@ -206,4 +214,141 @@ public function removeLeftItems($playlist)
...
+
+        $this->em->flush();
+
+        return true;
+    }
+
+    /**
+     * Clear playlists themes definitions unnasigning theme
+     *
+     * @param  [type] $theme          [description]
```
Comment (takeit) [inline newscoop/library/Newscoop/Services/PlaylistsService.php]: documentation block is wrong

**Comment by m038** [comment]
:+1:

**Pull request merged**